cegs_portal/search/views/v1/reg_effects.py

```diff
@@ -242,13 +251,13 @@ def get_data(self, options, feature_id) -> tuple[QuerySet[RegulatoryEffectObserv
                 feature_id, options.get("sig_only"), self.request.user.all_experiments()
             )
 
-        return reg_effects, feature
+        return reg_effects, feature, dna_feature
 
     def get_tsv(self, request, options, data, feature_id):
-        feature = DNAFeature.objects.get(accession_id=feature_id)
+        dna_feature = DNAFeature.objects.get(accession_id=feature_id)
```

Comment:
Huh. I'm curious why this was even here. Isn't this already in the "data" argument? I think you can take it out.